continuous_integration/recipe/meta.yaml

```diff
@@ -1,5 +1,4 @@
-{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev') + environ.get('VERSION_SUFFIX', '') %}
-{% set py_version=environ.get('CONDA_PY', 36) %}
+{% set version = environ.get('VERSION', '0.0.0.dev') + environ.get('VERSION_SUFFIX', '') %}
```

Comment:
For example using Jinja we could do something like [this]( https://github.com/conda-forge/cudatoolkit-feedstock/blob/5d6e3996ce74af5b1cdde677a6333ea665014aa4/recipe/meta.yaml#L446 )

Reply:
Thanks that example definitely looks much cleaner 🙂